### PR TITLE
Fix key error in Geometry features

### LINKED ISSUE #158: Error when trying to locate using co-ordinates
When trying to access endpoint `/locate/` the following error occurs:

KeyError at  locate

I'm working on the Nepalmap instance of Wazimap and trying to upgrade it to Python 3. I skimmed through the documentation but couldn't find much help (maybe I missed something). 

The function `get_locations_from_coords` (line 286) in `wazimap/geo.py` expects geometry features in following format:
``` 
{'properties': values, 'shape': values}
```
while currently it receives:
```
{'NP': {'properties': values, 'shape': values}}
```
and the above error occurs.

I edited the code in `geo.py` (line 286-291) as follows and it works fine.
```python
       for feature in features.values():
           code_value = list(feature.keys())[0]
           if feature['code_value']['shape'] and feature['code_value']['shape'].contains(p):
               geo = self.get_geography(feature['code_value']['properties']['code'],
                                        feature['code_value']['properties']['level'],
                                        version)
```

### PR DESCRIPTION
**Pull request by divyamani1**
Fixes #158

### CODE REVIEW

File: wazimap/geo.py [hunk 1]
```diff
@@ -285,9 +285,10 @@ def get_locations_from_coords(self, longitude, latitude, levels=None, version=No
 
         for features in self.geometry.values():
             for feature in features.values():
-                if feature['shape'] and feature['shape'].contains(p):
-                    geo = self.get_geography(feature['properties']['code'],
-                                             feature['properties']['level'],
+                feature_key = list(feature.keys())[0]
```
Comment (divyamani1) [inline wazimap/geo.py:288]: I was unsure of a variable name to use. Please suggest if any changes are to be made.

**Comment by divyamani1** [comment]
@ebsuku any updates on this?

**Comment by knightebsuku** [comment]
Hey @divyamani1 
Currently looking at this now.

File: wazimap/geo.py (hunk 1 above)
Comment (knightebsuku) [reply wazimap/geo.py:288]: That looks fine.

**Review by knightebsuku — APPROVED** [review]
OK, cool.
I just tried it for myself and it works

**Pull request merged**